### ISSUE TITLE
Topical Order Curriculum - Human Curated Situation Order

### ISSUE THREAD

**Issue by lichtefeld** [isi-vista/adam#986]
Expanding on a bullet point from #983 - We want to develop a robust software package that allows for modification of the situations presented to the language learner to better match data from actual human experiments. 

This robust software interaction would require:
[ ] A human-readable presentation of the situation which is easy to interpret quickly 
[ ] The ability to remove specific instances from the curriculum.
[ ] The ability to reorder specific instances from the curriculum,
[ ] The ability to load, save, and modify existing curriculum exports.
[ ] The ability to render the HTML curriculum dumps.
[ ] Bonus: GUI interface rather than command line for accessibility of future users?

This issue is waiting on #958 (which introduces the ability to pickle and unpickle curriculum) & completion of the randomly ordered curriculum for the intended experiment to use this 'topical' order to its curriculum.

**Comment by lichtefeld** [comment]
Closing as a topical curriculum generator for ADAM's symbolic scenes is not in scope for phase 3.
